src/sentry/tasks/deletion/hybrid_cloud.py

```diff
@@ -176,59 +171,62 @@ def _process_tombstone_reconciliation(
         watermark_manager: Manager = field.model.objects
         watermark_target = "r"
 
-    low, up, has_more, tid = chunk_watermark_batch(
-        prefix, field, watermark_manager, batch_size=get_batch_size()
-    )
-    to_delete_ids: List[int] = []
-    if low < up:
-        oldest_seen: datetime.datetime = timezone.now()
-
-        with connections[router.db_for_read(model)].cursor() as conn:
-            conn.execute(
-                f"""
-                SELECT r.id, t.created_at FROM {model._meta.db_table} r JOIN {tombstone_cls._meta.db_table} t
-                    ON t.table_name = %(table_name)s AND t.object_identifier = r.{field.name}
-                    WHERE {watermark_target}.id > %(low)s AND {watermark_target}.id <= %(up)s
-            """,
-                {
-                    "table_name": field.foreign_table_name,
-                    "low": low,
-                    "up": up,
-                },
-            )
-
-            for (row_id, tomb_created) in conn.fetchall():
-                to_delete_ids.append(row_id)
-                oldest_seen = min(oldest_seen, tomb_created)
+    with sentry_sdk.start_transaction(
+        op="deletion.hybrid_cloud.process_tombstone_reconciliation", name="process"
```

Comment:
```suggestion
        name="deletion.hybrid_cloud.process_tombstone_reconciliation", op="process"
```

Reply:
These are all essentially the same, right? I can't think why you'd want to differentiate tombstones from different models, so Mark's suggestion makes sense to me.